ikasaneip/recovery-manager/src/main/java/org/ikasan/recovery/ScheduledRecoveryManager.java

```diff
@@ -659,6 +661,10 @@ public void execute(JobExecutionContext context) throws JobExecutionException
             else
             {
                 this.consumer.start();
+                if(!isEventBaseRecovery)
+                {
+                    recoveryAttempts = 0;
```

Comment:
I would call cancelRecovery(null) as its a little cleaner. Also you should surely do the same in the above where the consumer is a scheduled consumer ?